### DISCUSSION TITLE
Is using `createUserWithEmailAndPassword` with Firebase JS SDK not possible?

### DISCUSSION THREAD

**Discussion by braincomb** [capawesome-team/capacitor-firebase#645] [General]
Hi,

We are considering adding `createUserWithEmailAndPassword` and `signInWithEmailAndPassword` to our Capacitor app on top of our social provider logins, but we currently have `skipNativeAuth=true` just so that we can use JS SDK to access Firestore, etc.

It appears that to use email/password sign-in method, `skipNativeAuth` must be `false`, which means we probably can no longer use Firebase JS SDK, which also means we will probably want to use `capawesome/firestore` plugin also. Is that correct?

Is there any workaround for this, since we would rather prefer to keep using Firebase JS SDK for accessing Firestore.

Thanks!

**Comment by robingenz** [comment]
You can set skipNativeAuth individually for each call: 

```ts
const createUserWithEmailAndPassword = async () => {
	await FirebaseAuthentication.createUserWithEmailAndPassword({
		skipNativeAuth: false,
		...
	});
};
```

Why don't you use the Firebase SDK directly for this call?

**Reply by braincomb** [reply]
Doesn't look like `createUserWithEmailAndPassword` supports `skipNativeAuth` as an option. It's only part of the `SignInOptions`.

```ts
export interface CreateUserWithEmailAndPasswordOptions {
    /**
     * @since 0.2.2
     */
    email: string;
    /**
     * @since 0.2.2
     */
    password: string;
}
```

> Why don't you use the Firebase SDK directly for this call?

Not ideal as it introduces a bit more complexity now that I have to mix-and-match two different APIs, but seems like I have no choice. Thanks for the suggestion!
